open-hackathon-server/src/hackathon/database/models.py

```diff
@@ -634,14 +634,18 @@ def __init__(self, **kwargs):
 class AzureKey(DBBase):
     """
     Azure certificate information of user/hackathon
+    Open-hackathon will try to use local certification file, if it doesn't exist, open-hackathon will try to
+    recover it from azure. If it runs in local environment, azure url actually equals local url.
     """
     __tablename__ = 'azure_key'
 
     id = Column(Integer, primary_key=True)
     # cert_url is cert file path in azure
     cert_url = Column(String(200))
-    # pem_url is pem file path in local
+    # pem_url is the local pem file path
     pem_url = Column(String(200))
+    # azure_pem_url is the pem file path in azure
+    azure_pem_url = Column(String(200))
```

Comment:
Do we have to add a new field? I'm not sure by curious why only one column designed `cert_url`? Is `pem_url` local and `azure_pem_url ` is remote url? If it is, I believe there must be some relationship between the two that means we can get one from the another?